src/components/common/Extensions/UserHub/partials/StakesTab/useStakesByFilterType.ts

```diff
@@ -35,23 +30,34 @@ export const useStakesByFilterType = () => {
     [data],
   );
 
-  const motionIds = useMemo(
+  const motionIdsMap = useMemo(
```

Comment:
This name should also be updated as the array represents motion stakes rather than motion IDs